addons/portal/static/src/chatter/frontend/attachment_upload_service_patch.js

```diff
@@ -5,8 +5,14 @@ import { patch } from "@web/core/utils/patch";
 patch(AttachmentUploadService.prototype, {
     _buildFormData(formData, file, thread, composer, tmpId, options) {
         super._buildFormData(...arguments);
-        if (thread.rpcParams.token && thread.model !== "discuss.channel") {
-            formData.append("token", thread.rpcParams.token);
+        if (thread.model !== "discuss.channel") {
```

Comment:
Do you still need this condition? channel shouldn't have hash/pid/token fields now?